docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1094,6 +1097,41 @@ Then when the FHIR server processes the POST requests for the `Encounter` and `P
 }
 ```
 
+#### 4.8.1.4 Example 5: Observation references Patient via relative local reference
+```
+{
+    "resourceType" : "Bundle",
+    "type" : "batch",
+    "entry" : [ {
+        "fullUrl" : "https://fhirserver1:9443/fhir-server/api/v4/Patient/new",
+        "resource" : {
+            "resourceType" : "Patient",
+            …
+        },
+        "request" : {
+            "method" : "POST",
+            "url" : "Patient"
+        }
+    }, {
+        "fullUrl" : "https://fhirserver1:9443/fhir-server/api/v4/Observation/new",
+        "resource" : {
+            "resourceType" : "Observation",
+            …
+            "subject" : {
+                "reference" : "Patient/new"
+            },
+            …
+        },
+        "request" : {
+            "method" : "POST",
+            "url" : "Observation"
+        }
+    } ]
+}
+```
+
+In example 5, the local identifiers for the request bundle entries are absolute URLs that conform to the FHIR specification's RESTful URL definition. When the FHIR server attempts to resolve the `Observation` request entry's `subject` reference, it will apply rule 6, since the reference is a relative URL. The local reference will be modified to be the original local reference (`Patient/new`) appended to the [root] from the `Observation` entry's local identifier (`https://fhirserver1:9443/fhir-server/api/v4/`). This will then be a valid reference  (`https://fhirserver1:9443/fhir-server/api/v4/Patient/new`) to the `Patient` request entry resource.
```

Comment:
could be brittle. Might be worth making `rule` a link.

Reply:
When you say local identifier do you mean `entry.fullUrl`?

Reply:
I removed references to numbered examples and rules, and instead created links as suggested, and referred to examples and rules with descriptive names. And yes, local identifier is synonymous with `entry.fullUrl`.